aws-rds-customdbengineversion/src/main/java/software/amazon/rds/customdbengineversion/CustomDBEngineVersionStatus.java

```diff
@@ -0,0 +1,60 @@
+package software.amazon.rds.customdbengineversion;
+
+import software.amazon.rds.common.status.StableStatus;
+import software.amazon.rds.common.status.TerminableStatus;
+
+public enum CustomDBEngineVersionStatus implements TerminableStatus, StableStatus {
+    Available("available", false, true),
```

Comment:
Nit:
I wish old good C style was a legitimate conventional approach in Java, like:
```java
Available("available", O_TERM | O_STABLE)
```

Maybe we can consider declaring an enum like `StatusOption` and use `EnumSet.of(...)` here instead.